sdk/tests/conformance2/renderbuffers/multisampled-renderbuffer-initialization.html

```diff
@@ -54,22 +50,32 @@
     // Set the clear color to green. It should never show up.
     gl.clearColor(0, 1, 0, 1);
 
-    runTest(gl, gl.canvas.width, gl.canvas.height, ALLOCATE_ATTACH);
-    runTest(gl, gl.canvas.width, gl.canvas.height, ATTACH_ALLOCATE);
-    runTest(gl, gl.canvas.width, gl.canvas.height, ALLOCATE_ATTACH_CLEAR_REALLOCATE);
-    runTest(gl, gl.canvas.width, gl.canvas.height, ALLOCATE_ATTACH);
-    runTest(gl, gl.canvas.width, gl.canvas.height, ATTACH_ALLOCATE);
-    runTest(gl, gl.canvas.width, gl.canvas.height, ALLOCATE_ATTACH_CLEAR_REALLOCATE);
+    let c = gl.canvas;
+    var maxSamples = gl.getInternalformatParameter(
+        gl.RENDERBUFFER, gl.RGBA8, gl.SAMPLES)[0];
+    for (let i = 0; i < 2; ++i) {
```

Comment:
Originally it did 2 tests, then it did the same 2 tests again. (It resizes the canvas between each test.) I don't know why, so I just put a loop around the whole thing.